vast.py

```diff
@@ -160,6 +171,15 @@ def load_permissions_from_file(file_path):
     with open(file_path, 'r') as file:
         return json.load(file)
 
+def complete_instance_machine(prefix=None, action=None, parser=None, parsed_args=None):
+  return show__instances(ARGS, {'internal': True, 'field': 'machine_id'})
```

Comment:
One consideration is that our API rate limiting policy is quite strict at the moment.  Users pressing tab for autocomplete could unknowingly trigger enough requests to get rate limited.

Reply:
correct. The *right* solution here is a caching layer. The XDG move allows us to instrument such an effort more properly but I made the decision that "1.0" shouldn't have a cache. Also, these are only run after the action is specified.

Reply:
Also last night I realized there should be a readme update as part of this task